ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -581,9 +581,9 @@ static const func_info_t func_infos[] = {
 	F1("ob_gzhandler",                          MAY_BE_FALSE | MAY_BE_STRING),
 
 	/* ext/hash */
-	F1("hash",                                  MAY_BE_FALSE | MAY_BE_STRING),
+	F1("hash",                                  MAY_BE_STRING),
 	F1("hash_file",                             MAY_BE_FALSE | MAY_BE_STRING),
-	F1("hash_hmac",                             MAY_BE_FALSE | MAY_BE_STRING),
```

Comment:
I am wondering why this line occurs here.. is there a difference in line endings?

Reply:
This line is replaced with the following line.